scroll-animations-1/Overview.bs

```diff
@@ -559,22 +587,20 @@ as follows:
 
     </div>
 
-    Note: Checking for {{endScrollOffset}} being the maximum scroll offset ensures that the common
+    Note: Checking for {{end}} being the maximum scroll offset ensures that the common
     case of a 'whole scroller' ScrollTimeline does not become inactive when you scroll to the end.
 
 1.  Return the result of evaluating the following expression:
 
     <blockquote>
-      <code>(<var>current scroll offset</var> - {{startScrollOffset}}) / ({{endScrollOffset}} - {{startScrollOffset}}) &times; [=effective time range=]</code>
+      <code>(<var>current scroll offset</var> - {{start}}) / ({{end}} - {{start}}) &times; [=effective time range=]</code>
     </blockquote>
 
-</div>  <!-- link-for-hint="ScrollTimeline" -->
-
-
 ## The 'animation-timeline' property ## {#animation-timeline}
 
 A {{ScrollTimeline}} may be applied to a CSS Animation [[CSS3-ANIMATIONS]] using
```

Comment:
I think this specification should mention how/if animation-timeline participates in animation shorthand property definition.

Reply:
Good point. Added a section to have this property participate in shorthand. At the moment this is kept in this spec but I expect it to become part of css-animations-2